packages/vincent-tools/vincent-tool-sdk/src/lib/toolCore/vincentTool.ts

```diff
@@ -89,18 +89,32 @@ export function createVincentTool<
         return wrapFailure(parsedToolParams);
       }
 
-      const result = await toolDef.execute(parsedToolParams, {
-        ...context,
-        policiesContext: { ...context.policiesContext, allow: true },
-      });
+      const result = await toolDef.execute(
+        // @ts-expect-error - TODO: fix this
```

Comment:
Let's merge this with this in place, since this is an internal method that isn't consumer-facing -- create a linear ticket to circle back and fix this type.